opm/simulators/wells/VFPProdProperties.cpp

```diff
@@ -161,7 +161,7 @@ EvalWell VFPProdProperties::bhp(const int table_id,
     EvalWell flo = detail::getFlo(table, aqua, liquid, vapour);
     EvalWell wfr = detail::getWFR(table, aqua, liquid, vapour);
     EvalWell gfr = detail::getGFR(table, aqua, liquid, vapour);
-    if (use_expvfp) {
+    if (use_expvfp || -flo.value() < table.getFloAxis().front()) {
```

Comment:
My concern here is that it is table specific. If the user creates a table with relatively big value for `table.getFloAxis().front()`, then we will use explicit for the region even with relatively big well rates. At the same time, with this approach, we do not need to invent a value to draw an invented line regarding whether we should use explicit ratios. It is okay to stay current way unless testing show the other way.

Reply:
But for that we can also see whether we need to do something with future testing.

Reply:
Yes, the feedback I got when I asked about this is that guidelines for creating tables say that there should be _sufficient_ amount of points in the low-flow region to accurately determine lowest possible rate (but with no guarantee that the converged solution should be above smallest value in table). So I think using explicit lookup below `table.getFloAxis().front()` is a very simple solution that also makes kind of sense (unless tests show otherwise).